apps/pod-issuer-server/src/routes/deel.ts

```diff
@@ -0,0 +1,115 @@
+import express, { Request, Response } from "express";
+import { expressjwt, Request as JWTRequest } from "express-jwt";
+import jwt from "jsonwebtoken";
+import { POD, PODEntries } from "@pcd/pod";
+import {
+  getDeelUserByEmail,
+  getPaystubPODByEmail,
+  savePaystubPOD
+} from "../stores/deel";
+
+const deel = express.Router();
+
+deel.post("/login", (req: Request, res: Response) => {
+  const inputs: { email: string; password: string } = req.body;
+  if (!inputs.email || !inputs.password) {
+    res.status(400).send("Missing query parameter");
+    return;
+  }
+
+  // In practice, get user information from database by email,
+  // Here for demo purposes, we'll allow any email with @zoo.com domain
+  if (!inputs.email.endsWith("@zoo.com")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // In practice, check if the encrypted password match
+  // This is just for demo purposes
+  if (!inputs.password.startsWith("zoo")) {
+    res.status(401).send("Invalid email or password");
+    return;
+  }
+
+  // Signing JWT, valid for 1 hour
+  const token = jwt.sign(
+    { email: inputs.email },
+    process.env.DEEL_EDDSA_PRIVATE_KEY!,
+    { algorithm: "HS512", expiresIn: "1h" }
+  );
+  res.send(token);
+});
+
+deel.post(
+  "/issue",
+  expressjwt({
+    secret: process.env.DEEL_EDDSA_PRIVATE_KEY!,
+    algorithms: ["HS512"]
+  }),
+  async (req: JWTRequest, res: Response) => {
+    const email = req.auth?.email;
+    if (!email) {
+      res.status(401).send("Unauthorized");
+      return;
+    }
+
+    const inputs: {
```

Comment:
Maybe comment that in a real scenario the user should have to prove they own the private keys of this semaphore commitment.